command.go

```diff
@@ -88,6 +88,9 @@ type Command struct {
 	// group commands.
 	Annotations map[string]string
 
+	//flagCompletionFunctions is map of flag completion functions.
+	flagCompletionFunctions map[*flag.Flag]func(cmd *Command, args []string, toComplete string) ([]string, ShellCompDirective)
```

Comment:
Could you move this a little lower to be with other non-public fields?

Reply:
Thanks for your response! 
Nice advice, I have moved `flagCompletionFunctions` after `globNormFunc`.